src/components/Tree/index.jsx

```diff
@@ -0,0 +1,193 @@
+import { Component } from 'react';
```

Comment:
I'm thinking of extracting this component into its own library and let other material users be able to use it. What do you think?

Reply:
Absolutely.